### PR TITLE
misc: Added Docker and GitHub meta stuff

### PR DESCRIPTION
**Pull request by favna**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Kyra
+/Skyra/ @kyranet
+/Skyra.Cache/ @kyranet
+/Skyra.Framework/ @kyranet
```
Comment (favna) [inline .github/CODEOWNERS]: this has to go

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+/Skyra.Gateway/ @kyranet
+/Skyra.Worker/ @kyranet
```
Comment (favna) [inline .github/CODEOWNERS]: this has to go

File: README.md [hunk 3]
```diff
@@ -1 +1,87 @@
...
+-   [`PostgreSQL`]: (Dev Optional) To store persistent data.
+-   [`Lavalink`]: (Dev Optional) Audio server.
+-   [`Redis`]: (Dev Optional) Audio server.
+<!-- -   [`InfluxDB`]: (Dev Optional) Analytics, Statistics, and Audit storage. -->
+
+## Set-Up
+
+Install a .NET IDE of your choice. We recommend in decending order:
+- JetBrains Rider
+- Visual Studio Code
```
Comment (favna) [inline README.md]:
```suggestion
- Microsoft Visual Studio
```

File: README.md [hunk 4]
```diff
@@ -1 +1,87 @@
...
+- Microsoft Visual Studio
```
Comment (favna) [inline README.md:22]:
```suggestion
- Visual Studio Code
```

**Review by favna — COMMENTED** [review]
Some changes!

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+**The issue tracker is only for issue reporting or proposals/suggestions. If you have a question, you can find us in our [Discord Server](https://join.skyra.pw)**.
+
+To contribute to this repository, feel free to create a new fork of the repository and
+submit a pull request.
+
+1. Fork, clone, and select the **master** branch.
+2. Create a new branch in your fork.
+3. Make your changes.
+4. Commit your changes, and push them.
+5. Submit a Pull Request [here](https://github.com/skyra-project/skyra/pulls)!
```
Comment (Tylertron1998) [inline .github/CONTRIBUTING.md]: you mean skyra-project/skyra-sharp?

**Review by Tylertron1998 — APPROVED** [review]
other then this, looks good to me

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (favna) [reply .github/CONTRIBUTING.md]: i cri. i thought i got em all

**Review by kyranet — APPROVED** [review]

**Pull request merged**